app/domain/authentication/authn_k8s/extract_container_name.rb

```diff
@@ -12,7 +12,6 @@ module AuthnK8s
       inputs: %i(service_id host_annotations)
     ) do
 
-      AUTHENTICATION_CONTAINER_NAME_ANNOTATION = "authentication-container-name"
       DEFAULT_AUTHENTICATION_CONTAINER_NAME = "authenticator"
```

Comment:
should this go in `consts.rb`?

Reply:
I don't think so, it isn't used anywhere outside this class.